### ISSUE TITLE
Share content with other wallabag users

### ISSUE THREAD

**Issue by Nimn** [wallabag/wallabag#679]
With a simple copy this should be fairly easy to implement but to prevent unsolicited content a mechanism to validate share between user should be developed. IMHO an efficient mechanism could be a user defined whitelist of user who can share content with.

**Comment by johngilden** [comment]
I agree that this would be an absolute killer feature. Imagine a team of people working on a research topic where everyone is just constantly adding new things to their "to-read" list that others might want to take a look at.

**Comment by jcharaoui** [comment]
Bookie, a similar piece of software, shares articles by default on its front page, unless a "private" setting is checked when creating the bookmark. Unfortunately, Bookie has pretty much ceased development, so I'd like to migrate to Wallabag and hopefully recover this feature in one form or another. Using a private URL to share a complete collection or tag seems to me like a really good compromise.

**Comment by nicosomb** [comment]
What is the URL of Bookie?

**Comment by tcitworld** [comment]
It's now dead. https://twitter.com/BookieBmarks

**Comment by jcharaoui** [comment]
@nicosomb The code is hosted here: https://github.com/bookieio/Bookie

**Comment by j0k3r** [comment]
Might be a good idea. We should put every new entry to private by default and user will have an option to set public by default for all new entry or switch it for one entry.

**Comment by Readon** [comment]
This definitely will be a killer feature that fits for group of user who is working together want to share knowledge with their own control. Some web page on internet will be removed permanently after a const time.

If it is possible, set the shared content to a group is better than all of them.

**Comment by tcitworld** [comment]
Related to #2268

**Comment by j0k3r** [comment]
> and should be easy to implement

Contributions are welcomed. The project is open source. 😉
